fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/identifiers/FedoraId.java

```diff
@@ -87,7 +87,7 @@
      * @throws IllegalArgumentException If ID does not start with expected prefix.
      */
     private FedoraId(final String fullId) {
-        this.fullId = ensurePrefix(fullId).replaceAll("/+$", "");
+        this.fullId = ensurePrefix(fullId).replaceAll("\\s", "%20").replaceAll("/+$", "");
```

Comment:
Perhaps there can be a FedoarId test for this behavior?

Reply:
I have reverted this change for a later attempt.